Modules/_sqlite/cursor.c

```diff
@@ -662,6 +662,17 @@ bind_parameters(pysqlite_state *state, pysqlite_Statement *self,
             return;
         }
         for (i = 0; i < num_params; i++) {
+            const char *name = sqlite3_bind_parameter_name(self->st, i+1);
+            if (name != NULL) {
+                PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
```

Comment:
If `PyErr_WarnFormat()` returns a value < 0, it sets an exception, and you should immediately return from the function.

Reply:
Thanks for the heads-up! I'll fix that.